src/stories/Autocomplete/Component/AdvancedParameters.stories.tsx

```diff
@@ -49,3 +50,21 @@ addComponentStoryDescription(
   `const args = ${stringifyWithDefaults(TermsWithGroupSuggestions.args)}`,
   termsWithGroupSuggestionsDescription
 );
+
+export const TermsWithFilters = ComponentTemplate.bind({});
+TermsWithFilters.args = {
+  apiKey,
+  onSubmit,
+  advancedParameters: {
+    autocompleteParameters: {
+      filters: {
+        'Products][color': 'Black',
```

Comment:
Can we use a different example for now? ][ action might confuse users.
We can 
* use group_id for filters because Search Suggestions have group IDs
* contact DS to add facets to Search Suggestions
* showcase a different parameter than filter

Reply:
would love it if we could provide an example that leverages group_ids as well as an example that leverages a different facet with standard/non-distracting key names

Reply:
Good points, opted to use a group_id filter instead.

I don't think any of the other autocomplete parameters currently make too much sense to show as an example so I stayed with filters.